lib/dfe/analytics.rb

```diff
@@ -186,6 +187,14 @@ def self.extract_model_attributes(model, attributes = nil)
       allowed_attributes.deep_merge(obfuscated_attributes.transform_values { |value| pseudonymise(value) })
     end
 
+    def self.add_ignored_columns_to_attributes(model, attributes)
+      ignored_columns = model.class.ignored_columns
+      return attributes unless ignored_columns && !ignored_columns.empty?
```

Comment:
I find it difficult to follow this unless and conditions. Would this be better:

```suggestion
      return attributes if ignored_columns.blank?
```

Reply:
now updated 👍

Reply:
@blank? works on `nil` anyway, so if if there was no check on `ignored_columns` it would still work :-)